openssl/src/ssl/mod.rs

```diff
@@ -967,8 +968,7 @@ impl SslContextBuilder {
 
     /// Appends a certificate to the certificate chain.
     ///
-    /// This chain should contain all certificates necessary to go from the certificate specified by
-    /// `set_certificate` to a trusted root.
+    /// Use `add_extra_chain_cert` to add the remainder of the certificate chain.
```

Comment:
I don't think this sentence makes sense on this method.

Reply:
yes agree, my bad again